oracle/tests/test_config.py

```diff
@@ -76,3 +78,14 @@ def test_check_misconfig_invalid_truststore_type(dd_run_check, instance):
     check = Oracle(CHECK_NAME, {}, [instance])
     with pytest.raises(Exception):
         dd_run_check(check)
+
+
+@patch('datadog_checks.oracle.oracle.PY2', True)
+def test_py2(dd_run_check, instance):
+    # Test to ensure that a ConfigurationError is raised when running with Python 2.
+    try:
```

Comment:
Small nit, but this test can be reformatted to be something like:

```
    with pytest.raises(ConfigurationError, match="This version of the integration is only available when using py3'"):
        check = Oracle('oracle', {}, [instance])
        dd_run_check(check)
```

Reply:
Is there a reason we want one over the other? Just want to understand what's preferred and why 🙇‍♂️

Reply:
I think it's just cleaner and more in line with how other tests for exceptions are formatted in the repo.

Reply:
👍 made the change.